src/FNAPlatform/FNAWindow.cs

```diff
@@ -149,15 +146,15 @@ internal void INTERNAL_ScreenDeviceNameChanged()
 			OnScreenDeviceNameChanged();
 		}
 
+		internal void INTERNAL_OnOrientationChanged()
+		{
+			OnOrientationChanged();
+		}
+
 		#endregion
 
 		#region Protected GameWindow Methods
 
-		protected internal override void SetSupportedOrientations(DisplayOrientation orientations)
```

Comment:
Just realized that this was here - sadly we have to keep this per the spec:

https://docs.microsoft.com/en-us/previous-versions/windows/xna/ff634564%28v%3dxnagamestudio.40%29

This can be another FNALoggerEXT line, I think.